jenkins-pipelines/longevity-1tb-7days-1Dis-2NonDis-Nemesises.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
```

Comment:
how's this pipe related to the refactoring ?

Reply:
The refactor is done for this pipeline :)